Palladium Rifts by Grinning Gecko/src/js/armor.js

```diff
@@ -32,6 +34,16 @@ async function applyDamageToNextActiveArmor(section, damage) {
   await setAttrsAsync(attrs);
 }
 
+on("change:repeating_armor:mvmt", async (e) => {
+  console.log("change:repeating_armor:mvmt", e);
+  const penalty = e.target.value;
```

Comment:
```suggestion
  const penalty = +e.target.value;
```

Reply:
Oops. Previous comment not right. Should be:

```suggestion
  const penalty = +e.newValue;
```

See https://wiki.roll20.net/Sheet_Worker_Scripts#change:.3Cattribute_name.3E .